ea_init_coregmrpopup.m

```diff
@@ -1,4 +1,19 @@
 function ea_init_coregmrpopup(handles,refine)
+% 
```

Comment:
Please provide a description of the function

Reply:
```
% Initialize coregistration methods popupmenu.
%
% INPUTS:
%    handles: LEAD GUI handle
%    refine: deprecated.
```